ml-agents/mlagents/trainers/tf_policy.py

```diff
@@ -172,7 +178,7 @@ def make_empty_memory(self, num_agents):
         :param num_agents: Number of agents.
         :return: Numpy array of zeros.
         """
-        return np.zeros((num_agents, self.m_size))
+        return np.zeros((num_agents, self.m_size), dtype=np.float)
```

Comment:
Out of scope for this PR, but should we try to make sure we never create float64 nparrays?